starboard/common/thread.cc

```diff
@@ -30,14 +30,16 @@ namespace starboard {
 struct Thread::Data {
   std::string name_;
   pthread_t thread_ = 0;
-  atomic_bool started_;
-  atomic_bool join_called_;
+  std::atomic_bool started_;
```

Comment:
Should these here be default-initialized just to be sure ? 
```
std::atomic_bool started_ = false
std::atomic_bool join_called_  = false;
```

Reply:
Done